api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -55,8 +55,6 @@ DbUser createUser(
   // Registers that a user has agreed to a given version of the Terms of Service.
   void submitTermsOfService(DbUser dbUser, Integer tosVersion);
 
-  void setDataUserCodeOfConductNameOutOfDate(String newGivenName, String newFamilyName);
```

Comment:
This is now a simple operation.  Also: we were never resetting this value!  Now we do, plus a test.